messaging/src/main/java/org/axonframework/commandhandling/configuration/CommandHandlingModule.java

```diff
@@ -72,24 +61,23 @@
  * @author Steven van Beelen
  * @since 5.0.0
  */
-public interface StatefulCommandHandlingModule extends Module, ModuleBuilder<StatefulCommandHandlingModule> {
+public interface CommandHandlingModule extends Module, ModuleBuilder<CommandHandlingModule> {
```

Comment:
I think that in a future PR, this interface can benefit from having a generic decorator option, just as you added for the `EventHandlingComponent`! :-)
Not for now, of course. I think this change will come naturally with the introduction of message interceptors.